helm-chart/binderhub/values.yaml

```diff
@@ -156,12 +156,15 @@ imageCleaner:
 
 ingress:
   enabled: false
+  https:
+    enabled: false
+    type: kube-lego
   hosts:
     - chart-example.local
   annotations: {}
     # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
-  tls: {}
+  tls: []
```

Comment:
The `ingress.tls` value was never referenced before this PR. It should be a `[]` though to match.